extension-requests/local-utils.js

```diff
@@ -110,3 +129,35 @@ function formDataToObject(formData) {
   }
   return result;
 }
+
+function dateDiff(date1, date2, formatter) {
+  if (date2 > date1) {
+    return dateDiff(date2, date1, formatter);
+  }
+
+  const timeDifference = new Date(date1).getTime() - new Date(date2).getTime();
+
+  const seconds = Math.floor(timeDifference / 1000);
+  const minutes = Math.floor(seconds / 60);
+  const hours = Math.floor(minutes / 60);
+  const days = Math.floor(hours / 24);
+  const months = Math.floor(days / 30);
+  const years = Math.floor(days / 365);
+
+  let res;
```

Comment:
Means?

Reply:
can you please provide more context? 

`res` usually means result of the function.

Reply:
Please provide a more meaningful name when refactoring.